electrode-ota-server-dao-plugin/src/dao.js

```diff
@@ -4,6 +4,10 @@ import UDTS from './models/UDTS.json';
 
 const historySort = history => history && history.sort((a, b) => b.created_.getTime() - a.created_.getTime());
 
+const releaseSort = (packageList) => {
+    return packageList.sort((a, b) => b.lastUpdated.getTime() - a.lastUpdated.getTime());
```

Comment:
Do we want to use lastUpdated for releaseSort?   If they updated an old package, that package will be the one release, no?

Reply:
The sort was intended to bubble up the ones where either the rollout value change or the tags change is newest.  I think lastUpdated is getting changed on any update to the record though.  I can change the updatePackage method to only update the timestamp when one of those values is different.  I'm also open to suggestions. :-)

Reply:
After looking over this some more, I agree, it should most likely sort by create time instead of lastUpdated.  I've pushed a change for this.